tests/issues/issue-58/issue-58.test.ts

```diff
@@ -0,0 +1,14 @@
+import { readFileSync } from 'fs';
+import { resolve } from 'path';
+import { format } from 'prettier';
+import { plugin } from './../../../src/index';
+
+describe('Issues', () => {
+	test('attribute interpolation using quotes', () => {
```

Comment:
I try to use sentences like "it should ..."
sadly jest uses the `test` function instead of `it`
Could you try to reformulate the sentence to e.g. `should interpolate attribute using quotes`? :slightly_smiling_face:

Reply:
@Shinigami92 AFAIK, `it` is also available in jest.

Reply:
Yeah, but only as an alias (https://jestjs.io/docs/en/api#testname-fn-timeout)
I think the only reason was to stick with non-aliased / native jest functions so users can find documentation more quickly and are not confused because they don't know other test tools like mocha